trunk/problems/LASER/UW_COUPLED/exec_case.F90

```diff
@@ -41,30 +41,30 @@ subroutine exec_case(idec)
 !
 !  ...Paraview graphics
       case(3)
-!     ...iParAttr parameter specifies which fields we dump out to paraview:
+!     ...iPvAttr parameter specifies which fields we dump out to paraview:
 !        - 1 H1 (heat)
 !        - 2 H(curl) (signal, E and H flux)
 !        - 2 H(curl) (pump,   E and H flux)
 !        - 1 H(div) (heat flux)
 !        - 6 L2 (signal, E and H field)
 !        - 6 L2 (pump,   E and H Field)
 !
-         iParAttr = (/1,2,2,1,6,6/)
```

Comment:
Why was this changed? I think it was better the other way, I typically view the L2 fields (they typically carry the most physical information) but they are often grouped. I already have to play games to reduce the file size, I don't want to export 6 files (or more for elasticity) when I only need one. If anything I'd go the other direction and allow specifying exactly which components to export